strings/java_substring_comparisons/Solution.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+import java.lang.String;
+
+public class Solution {
+  
+  public static String getSmallestAndLargest(String inputWord, int coefficient) {
```

Comment:
This seems like two different functions fused together, I think it's better to have getSmallestSubstringOfLength and getLargestSubstringOfLength. Also consider case when substring length is bigger then input's